### ISSUE TITLE
Test suite takes a long time to shut down

### ISSUE THREAD

**Issue by jamesnetherton** [quarkiverse/quarkus-google-cloud-services#586]
Not sure what's going on here, but it's taking 6+ minutes for the application under test to stop.

```
INFO  [io.quarkus] (main) quarkus-google-cloud-services-main-it stopped in 360.024s
```

**Comment by loicmathieu** [comment]
Juts noticed that, it seems to be related to Cloud Logging.

**Comment by jfbenckhuijsen** [comment]
@loicmathieu We can close this based on merging #777

**Comment by loicmathieu** [comment]
Yes, fixed by https://github.com/quarkiverse/quarkus-google-cloud-services/pull/777

By the way, better to include the issue number in the PR, I didn't remember all open issues ;) 
I usually even use Github special syntax to close them automatically.
